docs/blockchain/contracts/nft-contract/nft-actions/mknftofr.a.experimental.md

```diff
@@ -0,0 +1,78 @@
+---
+title: 'mknftofr.a'
+order: 37
+
+---
+
+# mknftofr.a
+
+Make an offer on a given token.
+
+## Technical Behavior
+
+The offer should be done at the time which is in the range of trading window of the NFT factory from which the NFT was issued.
+
+The action stores the offer to `nftoffer.a` table with the specified arguments. The new offer ID is read from `next.nftofr` table whose value field is then incremented.
```

Comment:
... value field is then incremented
should be
... `value` field is then incremented

Reply:
updated